src/rez/cli/env.py

```diff
@@ -116,6 +116,10 @@ def setup_parser(parser, completions=False):
     parser.add_argument(
         "--no-pkg-cache", action="store_true",
         help="Disable package caching")
+    parser.add_argument(
+        "--pkg-cache-mode", choices=["sync", "async"],
+        help="Optionally disable for force enable asynchronous package caching.  "
```

Comment:
This sentence is really hard to read. I'm not even sure it's a valid English sentence.

Reply:
Ha! Okay, try now.